SshAgentLib/PpkFormatter.cs

```diff
@@ -109,7 +122,8 @@ public class PpkFormatter : KeyFormatter
     internal enum Version
     {
       V1,
-      V2
+      V2,
+      V3
```

Comment:
Trailing comma would be nice

```suggestion
      V3,
```